doc/Language/glossary.pod6

```diff
@@ -4,8 +4,8 @@
 
 =SUBTITLE Glossary of Raku terminology
 
-X<|Abstract Class>
 =head1 Abstract class
```

Comment:
Sorry, been there, done that. Anchors must always be on top of the word, or the word itself it defines will be hidden. They should stay on top.

Reply:
Done.